invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/DescriptionsField/components/AdditionalDescriptionsField.js

```diff
@@ -15,79 +15,116 @@ import { emptyAdditionalDescription } from "./initialValues";
 import { i18next } from "@translations/invenio_rdm_records/i18next";
 import { sortOptions } from "../../../utils";
+import Overridable from "react-overridable";
 
 export class AdditionalDescriptionsField extends Component {
   render() {
     const { fieldPath, options, recordUI, editorConfig } = this.props;
+    const typeOptions = sortOptions(options.type);
+
     return (
-      <ArrayField
-        addButtonLabel={i18next.t("Add description")}
+      <Overridable
+        id="InvenioRdmRecords.DescriptionsField.Additional.container"
         defaultNewValue={emptyAdditionalDescription}
-        fieldPath={fieldPath}
-        className="additional-descriptions"
+        options={options}
+        typeOptions={typeOptions}
+        recordUI={recordUI}
+        editorConfig={editorConfig}
       >
-        {({ arrayHelpers, indexPath }) => {
-          const fieldPathPrefix = `${fieldPath}.${indexPath}`;
+        <ArrayField
+          addButtonLabel={i18next.t("Add description")}
+          defaultNewValue={emptyAdditionalDescription}
+          fieldPath={fieldPath}
+          className="additional-descriptions"
+        >
+          {({ arrayHelpers, indexPath }) => {
+            const fieldPathPrefix = `${fieldPath}.${indexPath}`;
+
+            return (
+              <Overridable
+                id="InvenioRdmRecords.DescriptionsField.Additional.grid"
+                editorConfig={editorConfig}
+                typeOptions={typeOptions}
+                recordUI={recordUI}
+              >
+                <Grid className="description">
+                  <Grid.Row>
+                    <Grid.Column mobile={16} tablet={10} computer={12}>
+                      <Overridable
+                        id="InvenioRdmRecords.DescriptionsField.Additional.field"
+                        editorConfig={editorConfig}
+                      >
+                        <RichInputField
+                          fieldPath={`${fieldPathPrefix}.description`}
+                          label={i18next.t("Additional Description")}
+                          editorConfig={editorConfig}
+                          optimized
+                          required
+                        />
+                      </Overridable>
+                    </Grid.Column>
+                    <Grid.Column mobile={16} tablet={6} computer={4}>
+                      <Overridable id="InvenioRdmRecords.DescriptionsField.Additional.remove">
+                        <Form.Field>
+                          <Button
+                            aria-label={i18next.t("Remove field")}
+                            className="close-btn"
+                            floated="right"
+                            icon
+                            onClick={() => arrayHelpers.remove(indexPath)}
+                          >
+                            <Overridable id="InvenioRdmRecords.DescriptionsField.Additional.remove.icon">
+                              <Icon name="close" />
```

Comment:
I see the usecase, but why not just give a className here and override it using css?

Reply:
agreed, I think we are going too granular with the overridables here